sdk/src/main/java/com/akkaserverless/javasdk/action/Action.java

```diff
@@ -34,7 +40,37 @@ protected final ActionContext actionContext() {
   }
 
   /** INTERNAL API */
-  public final void setActionContext(Optional<ActionContext> context) {
+  public final void _internalSetActionContext(Optional<ActionContext> context) {
     actionContext = context;
   }
+
+  public final Effect.Builder effects() {
+    return ActionEffectImpl.Builder$.MODULE$;
```

Comment:
could we add a method that is accessible from Java? Just looks foreign for any Java devs looking at this.